### ISSUE TITLE
Prometheus metrics endpoint

### ISSUE THREAD

**Issue by leogaggl** [helium/gwmp-mux#5]
Hey @lthiery,

More as a feature suggestion as an issue - are there any plans to expose some Prometheus endpoints for this?

It would be highly useful to allow metrics and alerts to be configured.

Cheers,
Leo

**Comment by lthiery** [comment]
No concrete plans but it should be not too difficult. I've used [this library](https://docs.rs/prometheus/latest/prometheus/) in other projects such as in [virtual-lorawan-device](https://github.com/helium/virtual-lorawan-device) and I think it would drop into here just fine too.

Can you detail what metrics you would like tracked?

**Comment by leogaggl** [comment]
Yeah. I have seen it used elsewhere, so I hope it's easy enough. The main thing would be throughput to each configured MUX client and anything that allows you to pick up any discrepancies between the two. No clients connected.
Breakdown by GatewayEUI or frequency might be interesting but not needed.

Thanks for the link to your Virtual Lora Device project. That is perfect timing while I am doing a lot of LNS and integration testing! Might even write some tests into my integrations for that! Nice work! 🙏
